app/src/main/java/agency/five/codebase/android/movieapp/ui/favorites/mapper/FavoritesMapperImpl.kt

```diff
@@ -0,0 +1,12 @@
+package agency.five.codebase.android.movieapp.ui.favorites.mapper
+
+import agency.five.codebase.android.movieapp.model.Movie
+import agency.five.codebase.android.movieapp.ui.component.MovieCardViewState
+import agency.five.codebase.android.movieapp.ui.favorites.FavoritesMovieViewState
+import agency.five.codebase.android.movieapp.ui.favorites.FavoritesViewState
+
+class FavoritesMapperImpl : FavoritesMapper {
+    override fun toFavoritesViewState(favoriteMovies: List<Movie>): FavoritesViewState {
+        return FavoritesViewState(favoriteMovies.map { FavoritesMovieViewState(MovieCardViewState(imageUrl = it.imageUrl, movieId = it.id, isFavorite = it.isFavorite)) })
```

Comment:
Please reformat this into multiple lines, to make it more readable. Since it is complex expression, I would suggest as much lines as possible, i.e. parameters of `MovieCardViewState` in separate lines, `MovieCardViewState` in new line compared to `FavoritesMovieViewState` constructor, everything in new line inside `map` operator, and whole mapping in new line compared to `FavoritesViewState`.